superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -65,8 +67,25 @@ def _init_distributed_setting(self):
                         ' distributed implementation: {}.'.format(self._name, self._args.distributed_impl)
                     )
                     return False
-
-                torch.distributed.init_process_group(backend=self._args.distributed_backend.value)
+                try:
+                    from torch.distributed import elastic    # noqa F401
```

Comment:
Shall we don't add `#noqa F401`

Reply:
if not add it, it can not pass lint for 'F401: import but not used'

Reply:
I think you should import it in the beginning, not in this code.